content/dogepedia/articles/how-can-my-business-accept-dogecoin.md

```diff
@@ -13,8 +13,10 @@ At the most foundational level, for your business to accept Dogecoin in exchange
 
 If you have a bricks-and-mortar business where payments are made in person, it’s common for merchants to print a QR code that is scanned by customers’ smartphones to streamline the transaction process.  
 
-If you have an ecommerce store, companies like Coinbase Commerce and BitPay make accepting Dogecoin a breeze, and include integrations with popular platforms like Shopify, WooCommerce, and Opencard!
+If you have an ecommerce store, companies like Coinbase Commerce and BitPay make accepting Dogecoin a breeze, and include integrations with popular platforms like Shopify, WooCommerce, and OpenCart. It is also possible to develop custom solutions connecting directly to a [Dogecoin Core node](/dogepedia/how-tos/operating-a-node/) through the JSON-RPC interface. The Dogecoin Foundation is currently developing [Libdogecoin](https://foundation.dogecoin.com/trailmap/libdogecoin/), a library that should, among other things, facilitate and standardize the implementation of Dogecoin payments.

```

Comment:
Perhaps change the word "should" to the word "will"?

Reply:
Changed!